docs/book/usage.md

```diff
@@ -83,6 +77,9 @@ return $vm;
 
 ## The DbSelect adapter
 
+> **Deprecated** since version 2.10.0.
+> Please use the [laminas/laminas-paginator-adapter-laminasdb](https://docs.laminas.dev/laminas-paginator-adapter-laminasdb) package if you wish to use the laminas-db-based pagination adapters.
```

Comment:
```suggestion
> ### Deprecated
>
> The adapter is deprecated since version 2.10.0. Please use the [laminas/laminas-paginator-adapter-laminasdb](https://docs.laminas.dev/laminas-paginator-adapter-laminasdb) package if you wish to use the laminas-db-based pagination adapters.
```

Reply:
For consistency with other documentations and notes of this kind.